src/utils/device.js

```diff
@@ -0,0 +1,69 @@
+/**
+* Returns an object with detected device and browser.
```

Comment:
Yes, if we don't have any crossbrowser bugs in other components, it must be simple.